metacat-main/src/main/java/com/netflix/metacat/main/api/v1/MetacatController.java

```diff
@@ -1035,7 +1047,7 @@ public TableDto updateMView(
         }
     )
     @Override
-    public TableDto updateTable(
+    public TableDto  updateTable(
```

Comment:
Why is there a double-spacing? Do you have the right code formatter?

Reply:
Should be single-space. I will change it.